files/zh-cn/web/html/element/heading_elements/index.html

```diff
@@ -165,7 +165,7 @@ <h4 id="嵌套">嵌套</h4>
  </li>
 </ol>
 
-<p>When headings are nested, heading levels may be "skipped" when closing a subsection.</p>
+<p>当小节嵌套时，若在子标签关闭前关闭母标签，则子标签也会一同关闭。</p>
```

Comment:
A suggestion, may be the misunderstanding:

```suggestion
<p>当小节中嵌套子标题时，若关闭了某个小节，则其中的嵌套的子标题可能会被“跳过”。</p>
```